packages/preset-rules/src/a11yConfig.ts

```diff
@@ -33,6 +33,10 @@ export function getA11yConfig(rules: string[]): A11yConfig {
             type: 'rule',
             values: rules,
         },
+        // Disable preloading assets as it causes timeouts for audio/video elements
+        //  with jest and delays webdriver tests by 2-3x when assets are not found (404)
+        //  Ref: https://github.com/dequelabs/axe-core/issues/2528
+        preload: false,
```

Comment:
@trevor-bliss @cordeliadillon This one line is essentially the fix for audio/video timeout issue. Rest of the changes are updating tests, docs and bumping versions.

Can you please review this when possible. I will get a release to npm with this fix once the PR is merged.